### ISSUE TITLE
Using key does not remove bypass button

### ISSUE THREAD

**Issue by paulkim26** [paulkim26/collab-museum#253]
Repro:
1. Spawn
2. Use key on history section lock
3. Can click on invisible bypass button

Pictured: history section entrance

Using the invisible bypass button will also spawn a new key

![image](https://github.com/paulkim26/collab-museum/assets/45007945/795ca71b-bb57-4622-82ff-6bba972bce51)

![image](https://github.com/paulkim26/collab-museum/assets/45007945/d0265700-2826-4219-ba2d-b755516bd5d9)

![image](https://github.com/paulkim26/collab-museum/assets/45007945/1a8160c2-066d-48be-a889-466191ca0dea)

**Comment by FranchNotFrench** [comment]
Oh! That's why the key On Exit trigger was linked to the prior Viz activators for the Lock Bypass Transparent. :D Good catch. It must have been a toggle before or something. I removed the linkage because it was actually enabling the Bypass transparent on unlock which I had already turned off, which took me like 10 minutes to sort out. But, I can just have each of the key slots disable the collider on the bypass button.
